packages/orm/src/EntityManager.ts

```diff
@@ -2227,6 +2220,23 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW, TX ext
 
     return result as any;
   }
+
+  /** Creates a new `type` and marks it as loaded, i.e. we know its collections are all safe to access in memory. */
+  #doCreate<T extends EntityW>(type: EntityConstructor<T>, opts: any, partial: boolean): T {
+    // The constructor will run setOpts which handles defaulting collections to the right state.
```

Comment:
question: this comment seems incorrect now since `setOpts` is being called below this?

Reply:
Ah yep, thought I removed that, but must have missed it